src/lib/cache.js

```diff
@@ -1,24 +1,67 @@
 import { isNull, isArray } from "lodash"
-import { Client } from "memjs"
 import config from "config"
-import { error } from "./loggers"
+import { error, verbose } from "./loggers"
+import redis from "redis"
+import url from "url"
 
-const { NODE_ENV } = process.env
+const { NODE_ENV, OPENREDIS_URL } = process.env
 const { CACHE_LIFETIME_IN_SECONDS } = config
 
-const createClient = () => {
-  if (NODE_ENV === "test") {
-    const store = {}
-    return {
-      store,
-      get: (key, cb) => cb(null, store[key]),
-      set: (key, data) => (store[key] = data),
-    }
+const isTest = NODE_ENV === "test"
+
+const VerboseEvents = ["connect", "ready", "reconnecting", "end", "warning"]
+
+function createMockClient() {
+  const store = {}
+  return {
+    store,
+    get: (key, cb) => cb(null, store[key]),
+    set: (key, data) => (store[key] = data),
+  }
+}
+
+function createRedisClient() {
+  const redisURL = url.parse(OPENREDIS_URL)
+  const client = redis.createClient({
+    host: redisURL.hostname,
+    port: redisURL.port,
+    retryStrategy: options => {
```

Comment:
in the docs this is snake case, `retry_strategy`.

Reply:
Good reviewing! 🕵️  At the bottom of https://github.com/NodeRedis/node_redis#usage-example they mention that all APIs/options can be either snake or camel case, so I went with camel because that‘s what we’ve been settling on for JS syntax style.